### PR TITLE
replace gtm code

### PR DESCRIPTION
**Pull request by rolandpo**
replace gtm library with regular tags

### CODE REVIEW

File: eagleproject/eagleproject/settings.py [hunk 1]
```diff
@@ -27,12 +27,9 @@
     DEBUG=(bool, False),
     DATABASE_HOST_OVERRIDE=(str, None),
     SENTRY_DSN=(str, None),
-    APP_ENV=(str, "local"),
 )
-if env("APP_ENV") != "staging":
-    # Staging is on heroku and doesn't use dotenv
-    env_file = Path(__file__).resolve().parent.joinpath(".env")
-    environ.Env.read_env(env.str("ENV_PATH", str(env_file)))
+env_file = Path(__file__).resolve().parent.joinpath(".env")
```
Comment (shahthepro) [inline eagleproject/eagleproject/settings.py:31]: Can you revert back this change? It can probably break prod environment

File: eagleproject/eagleproject/settings.py [hunk 2]
```diff
@@ -74,8 +71,6 @@
     "abi",
     "core",
     "notify",
-    "gtm",
-    "django_extensions",
```
Comment (shahthepro) [inline eagleproject/eagleproject/settings.py:78]: Please, revert these changes as well

**Review by shahthepro — CHANGES_REQUESTED** [review]
@rolandpo Looks like your branch reverts a lot of changes on prod. Can you pull master and do your changes there?

**Comment by rolandpo** [comment]
sorry my bad, should be fixed now

**Review by shahthepro — APPROVED** [review]
